apps/mobile/src/navigation/MainTabNavigator/TabBar.tsx

```diff
@@ -20,25 +22,37 @@ type TabItemProps = {
   navigation: MaterialTopTabBarProps['navigation'];
 };
 
+const TAB_NAMES = {
```

Comment:
We should add a type here so we never forget to keep this updated. 

Something like this
```ts
const TAB_NAMES: { [key in keyof MainTabNavigatorParams]: string } = {...mappingsHere}
```